molecule/aap-test/molecule.yml

```diff
@@ -0,0 +1,24 @@
+---
+dependency:
+  name: galaxy
+  options:
+    requirements-file: molecule/requirements.yml
+driver:
+  name: default
+platforms:
+  - name: instance
+    private_key_file_path: ~/.ssh/id_rsa
+    image_id: ami-0cd93da2da7f10cab
```

Comment:
Let's also use `ami-0dac29b55bd7c33aa` throughout this PR please (RHEL 9.4, upgraded for the whole collection in https://github.com/securesign/artifact-signer-ansible/pull/139/files)